code/modules/psionics/faculties/coercion.dm

```diff
@@ -60,6 +60,18 @@
 		to_chat(user, SPAN_WARNING("\The [target] is in no state for a mind-read."))
 		return TRUE
 
+	var/obj/item/organ/internal/augment/psi/psiaug = locate() in target
```

Comment:
If we do find a psiaug, there is no check to make sure that this psiaug is implanted. Needs a check like on line 76.

Reply:
Actually it's an augment, do this `var/obj/item/organ/internal/augment/psi/psiaug = target.internal_organs_by_name[BP_AUG_PSI]`